### PR TITLE
Feat/solid mesh

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: src/geode/mesh/core/solid_mesh.cpp [hunk 1]
```diff
@@ -0,0 +1,1093 @@
...
+            Facets::overwrite( from );
+        }
+
+        void overwrite_edges( const Edges& from )
+        {
+            Edges::overwrite( from );
+        }
+
+    private:
+        void convert_attribute_to_abseil()
```
Comment (panquez) [inline src/geode/mesh/core/solid_mesh.cpp:326]: should allow downward compatiblity? really cool

File: src/geode/mesh/core/solid_mesh.cpp [hunk 2]
```diff
@@ -0,0 +1,1093 @@
...
+        return false;
+    }
+
+} // namespace
+
+namespace geode
+{
+    template < index_t dimension >
+    class SolidMesh< dimension >::Impl
+        : public detail::FacetStorage< PolyhedronFacetVertices >,
```
[the remark this answers is not in the document]
Comment (BotellaA) [reply src/geode/mesh/core/solid_mesh.cpp:153]: I left everything like it was regarding FacetStorage. I am not sure the potential perf improvement is forth the added complexity.

File: src/geode/mesh/core/solid_mesh.cpp (hunk 1 above)
Comment (BotellaA) [reply src/geode/mesh/core/solid_mesh.cpp:326]: that's not new. It is an old convertion

**Pull request merged**